docs/getting-started/quick-start-guide.md

```diff
@@ -44,9 +44,9 @@ A development server will be started. This server will be listening on http://lo
 
 Open the project directory in your preferred code editor/IDE and try editing some of the files under `packages/mars-theme`. Each time you save a change the browser will automatically reload and display the new version as these changes are detected by the development server.
 
-5 - **Set your own WordPress installation** as the source of data
+5 - **Set your own WordPress installation** as data source
```

Comment:
Replace
"**Set your own WordPress installation** as data source"
with
"**Set your own WordPress installation** as the data source"